python/CMakeLists.txt

```diff
@@ -37,6 +37,10 @@ endfunction()
 #
 file(STRINGS "requirements.txt" REQUIREMENTS REGEX "^[^#].+$")
 check_python_requirements(${REQUIREMENTS})
+install(
+  FILES "requirements.txt"
+  RENAME "revng-runtime-requirements.txt"
+  DESTINATION "${CMAKE_INSTALL_PREFIX}/share/revng/")
```

Comment:
Please drop this commit, once Python is in orchestra, we'll have two distinct components pulling in Python deps, one as a run-time dependency, one as a build-time dependency.

Reply:
Dropped